sources/packages/backend/libs/test-utils/src/factories/education-program.ts

```diff
@@ -18,7 +18,6 @@ export function createFakeEducationProgram(relations?: {
   program.credentialType = "credentialType";
   program.cipCode = "cipCode";
   program.nocCode = "nocCode";
-  program.sabcCode = "sabcCode";
```

Comment:
why it's removed?

Reply:
Please see https://github.com/bcgov/SIMS/pull/1750#discussion_r1119412225